ballerina/client.bal

```diff
@@ -52,7 +53,8 @@ public type Client client object {
     # + return - Metadata of the query execution as an `sql:ExecutionResult[]` or an `sql:Error`
     remote isolated function batchExecute(ParameterizedQuery[] sqlQueries) returns ExecutionResult[]|Error;
 
-    # Executes an SQL query, which calls a stored procedure. This may or may not return results.
+    # Executes an SQL query, which calls a stored procedure. This may or may not
+    # return results as a `sql:ProcedureCallResult` which should be closed after performing this operation.
```

Comment:
```suggestion
    # Executes an SQL query, which calls a stored procedure. This may or may not
    # return results. Once the results are processed, invoke the `close` method on the `sql:ProcedureCallResult` to release the connection resources.
```

Reply:
Alos lets add another not on `sql: ProcedureCallResult` close method